docs/algob-config.md

```diff
@@ -0,0 +1,51 @@
+# algob config
+
+Algorand builder project must have an `algob.config.js` file present in a root directory.
+The config is used to list available algorand networks and instructions how to connect to them.
+
+A network object can specify following entries:
+
++ `host` (required, default `none`)
++ `port` (required, default `"8080"`)
++ `token` (required, default `none`)
++ `postHeader` -- HTTP header attached to every raw transaction request (optional, default `none`)
+
+
+## Example
+
+```
+
+const {devnet} = require("algorand-builder/config")
+
+var mydevnet = {
+     host: "127.0.0.1",
+     port: "8080",
+     token: "abc",
+}
+
+
+module.exports = {
+  /**
+   * Networks define how you connect to your Algorand node. By default the
+   * `development` network is used.
+   */
+  networks: {
+    // You can run an a algorand node a separate terminal, or use `algob` to run and
+    // configure for you a devnet. If `development` network is not specified, `algob`
+    // will use the internal `devnet` configuration for development network.
+    development: mydevnet,
+    devnet: devnet,
```

Comment:
Is it a good idea to move out the config in the example? Advanced users will know for themselves.

Reply:
Are you talking about proper documentation? We will do a whole docs in the last two sprints.

Reply:
I thought that it's simpler not to have variables `mydevnet` and `devnet` and move the config into one blob.

Reply:
well, it's just a variable. Feel free to rename it in an example.